x-pack/plugins/security_solution/public/common/components/filter_group/index.tsx

```diff
@@ -95,13 +108,14 @@ const FilterGroupComponent = (props: PropsWithChildren<FilterGroupProps>) => {
   }, []);
 
   const onUrlParamInit = (param: FilterItemObj[] | null) => {
-    if (param == null) return;
+    if (!param) setInitialUrlParam([]);
     try {
-      setInitialUrlParam(param);
+      setInitialUrlParam(param ?? []);
```

Comment:
is that `if` statement necessary? line 113 would handle the edge case here I think. 

rn if param is null or undefined, `setInitialUrlParam` will be called twice (?)